src/pollution_model/run_pollution_model.py

```diff
@@ -0,0 +1,474 @@
+    SELECT building_id, geometry FROM nz_building_outlines
+    WHERE ST_INTERSECTS(nz_building_outlines.geometry, ST_GeomFromText('{aoi_wkt}', {crs}));
+    """
+    # Execute the SQL query
+    result = gpd.GeoDataFrame.from_postgis(query, engine, index_col="building_id", geom_col="geometry")
+    new_result = []
+
+    buildings = gpd.GeoDataFrame.from_file("central_buildings.geojson")
+    result = buildings
+    for index, item in result.iterrows():
+        # TODO: Update this when the real database has been created. For the time being, we use a geojson file.
+        # Append appropriate attribute data to the list. The attributes are Index, SurfaceArea, and SurfaceType.
+        # Additionally, a placeholder for TSS, TCu, TZn, DCu, and DZn are included and set to "None". These will be
+        # edited later.
+        surface_area = result.iloc[index]["geometry"].area
+        new_result.append({"Index": index, "SurfaceArea": surface_area,
+                           "SurfaceType": result.iloc[index]["surface_type"],
+                           "TSS": None, "TCu": None, "Tzn": None, "DCu": None, "DZn": None})
+
+    # return the GeoDataFrame containing the relevant data about buildings
+    return gpd.GeoDataFrame(new_result)
+
+
+def get_road_information(engine: Engine, area_of_interest: gpd.GeoDataFrame):
+    """
+    Extracts relevant information about roads and car parks from the database and formats them such that they are easy
+    to use for pollution modeling purposes.
+
+    Parameters
+    ----------
+    engine: Engine
+      The sqlalchemy database connection engine
+    area_of_interest : gpd.GeoDataFrame
+        A GeoDataFrame polygon specifying the area of interest to retrieve buildings in.
+    Returns
+    -------
+    xr.DataArray
+        A DataArray containing rows corresponding to roads, and columns corresponding to
+        attributes (Index, SurfaceArea, SurfaceType)
+    """
+    aoi_wkt = area_of_interest["geometry"][0].wkt
+    crs = area_of_interest.crs.to_epsg()
+
+    # Select all relevant information from the appropriate table
+    # TODO: Update this when the database has been created
+    query = f"""
+    SELECT road_id, geometry FROM nz_roads
+    WHERE ST_INTERSECTS(nz_roads.geometry, ST_GeomFromText('{aoi_wkt}', {crs}));
+    """
+
+    # Execute the SQL query
+    result = gpd.GeoDataFrame.from_postgis(query, engine, index_col="road_id", geom_col="geometry")
+    new_result = []
+
+    for index, row in result.iterrows():
+        # TODO: Update this when the real database has been created
+        # Append appropriate attribute data to the list. The attributes are Index, SurfaceArea, and SurfaceType.
+        # Additionally, a placeholder for TSS, TCu, TZn, DCu, and DZn are included and set to "None". These will be
+        # edited later.
+        # Calculate surface area of roads. Currently, an approximation based on the length of the road * 5
+        surface_area = row["geometry"].length*5

```

Comment:
I think this works the same as a buffer as a placeholder, since its a 2D line being made into an approximation of 3D

Reply:
> I think this works the same as a buffer as a placeholder, since its a 2D line being made into an approximation of 3D

It's just a placeholder for now, so it doesn't matter much, but knowing that the buffer option is available could be useful later on.